en-US/navigation.lang

```diff
@@ -303,3 +310,175 @@ Get a Firefox Account
 
 ;Check out the Benefits
 Check out the Benefits
+
+
+;Firefox Blog
+Firefox Blog
+
+
+;Read about new Firefox features, and get tips for staying safer online.
+Read about new Firefox features, and get tips for staying safer online.
+
+
+;Meet the technology company that puts people before profit.
+Meet the technology company that puts people before profit.
+
+
+;The non-profit behind Firefox is fighting for a healthy internet for all.
+The non-profit behind Firefox is fighting for a healthy internet for all.
+
+
+;Learn how Firefox treats your data with respect.
+Learn how Firefox treats your data with respect.
+
+
+;Mozilla Careers
+Mozilla Careers
+
+
+;Approach your career with a sense of purpose. Find worthy work at Mozilla.
+Approach your career with a sense of purpose. Find worthy work at Mozilla.
+
+
+;Browsers
+Browsers
+
+
+# Accessible label for the 'X' button that closes a menu
+;Close Browsers menu
+Close Browsers menu
+
+
+# please do not translate the product name "Firefox Browser"
+;Firefox Browser for Desktop
+Firefox Browser for Desktop
+
+
+;Get the browser that respects your privacy automatically. On Windows, MacOS or Linux.
+Get the browser that respects your privacy automatically. On Windows, MacOS or Linux.
+
+
+# please do not translate the product name "Firefox Browser"
+;Firefox Browser for Mobile
+Firefox Browser for Mobile
+
+
+;Take speed, privacy and peace of mind with you. On Android and iOS.
+Take speed, privacy and peace of mind with you. On Android and iOS.
+
+
+;iOS Support
+iOS Support
+
+
+;Android Support
+Android Support
+
+
+# please do not translate the product name "Firefox Browser"
+;Firefox Browser for Enterprise
+Firefox Browser for Enterprise
+
+
+;Get unmatched data protection on the release cadence that suits your organization.
+Get unmatched data protection on the release cadence that suits your organization.
+
+
+;Strap on your goggles and step into the immersive web with Firefox’s VR browser.
```

Comment:
Firefox as a brand must be left unchanged. This format "Firefox's" xyz will force some languages to declense the brand so it looks like Firefoxa or Firefoksa to be grammatically correct to name a few. This is for languages using Latin scripts. Can we change the wording to make it easier for localizer, like `VR browser of Firefox`. For other non-latin scripts, some face equally challenging situation.

Reply:
We've asked for a different version of this blurb, just waiting for that to be final.

Reply:
Updated!